sw/device/lib/crypto/impl/drbg.c

```diff
@@ -35,8 +36,16 @@ static status_t seed_material_construct(
   seed_material->len = nwords;
```

Comment:
i think we should check the length vs. allocated space [struct](https://github.com/lowRISC/opentitan/blob/7c8100d89f0de0b27bc63bd4b54a5bc576c0719c/sw/device/lib/crypto/drivers/entropy.h#L51C17-L51C34)

otherwise i think the user can supply a buffer that is too long and overwrite stack

Reply:
HARDENED_CHECK_EQ(seed_material.len, kEntropySeedWords); ?